docs/data-sources/vlan.md

```diff
@@ -29,16 +39,17 @@ data "netbox_vlan" "vlan2" {
 
 ### Optional
 
+- `group` (Number)
 - `name` (String) At least one of `name` or `vid` must be given.
```

Comment:
Can we get rid of the AtLeastOneOf for name and vid?

Reply:
I was pondering it myself originally, but wasn't sure what to do.

It is removed now and all schema attributes can be used freely.